src/OMSimulatorLib/StepSizeConfiguration.h

```diff
@@ -0,0 +1,49 @@
+#ifndef _OMS_STEP_SIZE_CONFIGURATION_H_
+#define _OMS_STEP_SIZE_CONFIGURATION_H_
+
+#include "SignalRef.h"
+
+#include <vector>
+
+//auxiliary data structures
+struct staticBound {
+    double lower;
```

Comment:
This file is using 4 spaces as indentation; it would be good to change it to 2 spaces, like we do for all other source files.